api/resolvers/wallet.js

```diff
@@ -308,7 +328,8 @@ export default {
 
   Invoice: {
     satsReceived: i => msatsToSats(i.msatsReceived),
-    satsRequested: i => msatsToSats(i.msatsRequested)
+    satsRequested: i => msatsToSats(i.msatsRequested),
+    expired: i => new Date(i.expiresAt) <= new Date()
```

Comment:
required to re-render component when invoice expired

Reply:
What?

Reply:
Without this, it will never show expired in the frontend since the component only renders again if the query actually returned something else.

Reply:
Ah, I see. I guess you could `setTimeout` on the frontend instead. Kind of weird to control rendering from the backend.

Come to think of it, this something we could add to our countdown component because it'd be nice to do this when someone is editing something and the edit window expires.

Reply:
I used `onComplete` from the `<Countdown>` component now